src/constants/stateToLabelMaps.ts

```
@@ -22,3 +30,43 @@ export const recruitmentStatusMap: Record<'true' | 'false', string> = {
   true: '모집 중',
   false: '모집 완료',
 }
+
+export const linkValueToLabelMap = linkOptions.reduce(
+  (acc, { value, label }) => ({
+    ...acc,
+    [value]: label,
+  }),
+  {} as Record<string, string>
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)